ckan/migration/versions/016_uuids_everywhere.py

```diff
@@ -166,19 +172,18 @@ def create_uuids(migrate_engine, primary_table_name, revision_table_name):
 
 def drop_sequencies(migrate_engine):
 
-    sequencies = ['package_extra', 'package_extra_revision', 'package',
-                  'package_resource', 'package_resource_revision',
-                  'package_revision',' package_tag', 'package_tag_revision',
+    sequencies = ['package_extra', 'package',
+                  'package_resource',
+                  'package_tag',
```

Comment:
looks like a change in behaviour. we'd still want to drop these sequences if they were created in an earlier migration, right?